packages/kbn-server-route-repository-utils/src/parse_endpoint.ts

```diff
@@ -20,9 +20,5 @@ export function parseEndpoint(endpoint: string) {
     throw new Error(`Endpoint ${endpoint} was not prefixed with a valid HTTP method`);
   }
 
-  if (!version && pathname.startsWith('/api')) {
```

Comment:
we can't infer this from the endpoint anymore, given that there is a server-side option that can override (our own) semantics of /api vs /internal

Reply:
Perhaps we can check if the access setting is public and enforce this check in that case? 
Because the default is internal.

Reply:
this check is used in the client, so no bueno